src/plugin/weekOfYear/index.js

```diff
@@ -6,13 +6,27 @@ export default (o, c, d) => {
     if (week !== null) {
       return this.add((week - this.week()) * 7, 'day')
     }
+
+    let weekStart = 0
+    const locale = this.$locale()
+    if (locale.weekStart) {
+      // eslint-disable-next-line prefer-destructuring
+      weekStart = locale.weekStart
+    }
+
```

Comment:
const weekStart = this.$locale().weekStart || 0

Reply:
Great! Already update.